docs/API/fliplet-notifications.md

```diff
@@ -105,10 +140,20 @@ In-app notifications can also notify the user with a push notification which can
 // insert an in-app notification and also send a push notification
 instance.insert({
   data: { message: 'Hi John!' },
+
+  // Also send a push notification
   pushNotification: {
     payload: {
       title: 'Title of the push notification',
-      body: 'Message of the push notification'
+      body: 'Message of the push notification',
+      badge: 1, // Set the notification badge number
+      custom: {
+        customData: { // Add a link to the push notification
```

Comment:
this comment should be before the actual `customData` key:

```
// Add a link to the push notification
customData: { }
```

Reply:
Updated